README.md

```diff
@@ -32,12 +32,9 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
-## Running the App
+## Preparing and running the App
```

Comment:
We should leave the instructions for how to run the app after the first time as well as these setup instructions - there's no advantage in forcing the user to run all of the setup steps every time they want to start the app
It might be worth being clear here as well, that this is an _alternative_ setup to the instructions above (which are still worth keeping, for users on other systems)